### ISSUE TITLE
Analysis report feedback

### ISSUE THREAD

**Issue by ugurdogrusoz** [nazliozge/CS319-OO-Software-Development-Project#1]
[8.analysis.pdf](https://github.com/nazliozge/CS319-OO-Software-Development-Project/files/182265/8.analysis.pdf)

**Comment by ugurdogrusoz** [comment]
[9.analysis.pdf](https://github.com/nazliozge/CS319-OO-Software-Development-Project/files/182377/9.analysis.pdf)

Please see me about your report sometime this week!

**Comment by nazliozge** [comment]
- Grammar mistakes are corrected
- "User" from Use-Case model is changed as "Player"
- Subsection naming is changed from "Class Diagrams" etc to meaningful names
- Class diagram is redrawn based on your feedback and all the related parts are updated
- State Diagram is redrawn as it was previously not correct
- General Activity Diagram and Overall Activity Diagram are redrawn
- All 6 Sequence Diagrams are revised based on the new class diagram
So, to sum up, Figure 35(Use-Case Model) has a minor change and from Figure 36 to Figure 45 all the diagrams are revised and redrawn.

**Comment by ugurdogrusoz** [comment]
I see a very recent analysis document which is a few pages longer. What's up with that?

**Comment by nazliozge** [comment]
The most recent change is actually not a change. It is just a misclick while uploading other classes ("Account and CoinWallet classes are added.") which resulted as a change but it is not, it is probably a blank space. The original and the final document which is FINAL.Analysis.Report.by.Group.9.pdf, is uploaded on the due date as the final revised version of the analysis report. And it is a few pages longer based on the last feedbacks we received and wrote above.

**Comment by ugurdogrusoz** [comment]
Nazli, the document named FINAL.Analysis.Report.by.Group.9.pdf was uploaded 6 hours ago (this morning)!?

**Comment by ugurdogrusoz** [comment]
I don't see any changes in your sequence and activity diagrams! Yet you claim you revised them?

**Comment by ArasHeper** [comment]
the "FINAL.Analysis.Report.by.Group.9.pdf" is the first version, Revised one seems to be the last

**Comment by ugurdogrusoz** [comment]
I just lost more than half an hour because you failed to properly provide me with the correct file name/URL for the revised analysis report!
